src/runtime/runtime.js

```diff
@@ -803,6 +813,14 @@ export class ConfiguredRuntime {
             error = 'Unknown skipAccountCreationScreen value: ' + value;
           }
           break;
+        case 'publisherProvidedId':
+          if (
+            value != undefined &&
+            !(typeof value === 'string' && value != '')
+          ) {
+            error = 'Unknown publisherProvidedId value: ' + value;
```

Comment:
If it must be a string, can we make the error message say that? (more helpful for developers)